src/modals/CssSnippetFuzzySuggestModal.ts

```diff
@@ -94,13 +123,20 @@ export class CssSnippetFuzzySuggestModal extends FuzzySuggestModal<
 					}
 				)
 			);
+			const isEnabled = this.getOrToggleSnippet(item.item)
+			if (isEnabled !== undefined) {
+				el.appendChild(createDiv(
+					{ cls: ["suggestion-aux", "css-editor-status", isEnabled ? "enabled" : "disabled"] },
+					(el) => el.appendText(isEnabled ? "enabled": "disabled")
+				))
+			}
 		}
 		if (this.inputEl.value.trim().length > 0 && item.match.score === 0) {
 			el.appendChild(
 				createDiv({ cls: "suggestion-aux" }, (el) => {
 					el.appendChild(
 						createSpan({ cls: "suggestion-hotkey" }, (el) => {
-							el.appendText("Enter to create");
+							el.appendText("Shift Enter to create");
```

Comment:
```suggestion
							el.appendText("Enter to create");
```

This change makes it no longer match the quick switcher, I think this should be left as is. Here's what the quick switcher looks like when typing something that doesn't exist.

<img width="739" alt="image" src="https://github.com/Zachatoo/obsidian-css-editor/assets/6936914/3364b77a-daed-4e0b-8359-9d53768c7660">

Reply:
But it's incorrect. You still need to use "shift enter" to create the file. It's not enough to just use enter to create the new file.

Reply:
I also thought that on Mac most (smaller) keyboards don't have the <code>del</code> key,  so I'll need to press <code>Fn Meta backspace</code>. Decided against changing it to just <code>Meta backspace</code> or similar, since this PR was about the state issue.

Reply:
It is enough to just use <kbd>Enter</kbd> to create a file, if you type something that shows no fuzzy suggestions (the "No X found" message is shown). This would be the only way for a mobile user to create a new file that doesn't exist. The quick switcher and the CSS editor quick switcher both work this way.

Reply:
Hmm... I was sure I tested this, and it didn't allow me to create it without using shift as in the normal selection mode. Now, when I retested it, it surely is enough with the Enter. So I'm going to change it back to that again. 🙈